server/routes/creatingLicences/handlers/checkAnswers.ts

```diff
@@ -27,11 +27,19 @@ export default class CheckAnswersRoutes {
       )
     }
 
-    const { conditionsWithUploads, additionalConditions } = this.conditionService.additionalConditionsCollection(
-      licence.additionalLicenceConditions
-    )
+    const conditionsToDisplay = await this.conditionService.getAdditionalAPConditionsForSummaryAndPdf(licence, user)
 
-    res.render('pages/create/checkAnswers', { additionalConditions, conditionsWithUploads, backLink })
+    const { conditionsWithUploads, additionalConditions } =
+      this.conditionService.additionalConditionsCollection(conditionsToDisplay)
+
+    const parentBespokeConditions = await this.conditionService.getbespokeConditionsForSummaryAndPdf(licence, user)
```

Comment:
This appears to have similar behaviour to the AP condition equivalent, which means if it's not appropriate to get the parentBespokeConditions, it will get the licence bespoke conditions instead. I don't think this is a problem, but I think it makes the naming a bit confusing. It also means that in the njk bespoke conditions macro, `parentBespokeConditions` will always be populated if `licence.bespokeConditions` is, meaning that the check on `licence.bespokeConditions` is somewhat unused, because either you will want the parent conditions (which will be in `parentBespokeConditions`, or you want the licence bespoke conditions, which will also be populated in `parentBespokeConditions`.

This isn't a blocker, but is a simplification to the logic that we could make.

Reply:
why I used parentBespokeConditions is I wanted to differentiate between existing bespokeConditions macro and my variable. As it is not recommended to use the same names for methods and variables. If you feel it need's to be changed please suggest me the name.

Other one is if we are using parentBespokeConditions in bespokeConditionsSummary page then licence.bespokeConditions is useless. Yes, it will be only when we are calling macro from checkAnswers.njk and viewActiveLicence.njk files and for remaining files it will only use licence.bespokeConditions as parentBespokeConditions will be null elsewhere.

Reply:
Ah, I hadn't realised the bespokeConditions macro was used elsewhere.
In which case, the only change I would possibly suggest would be changing `parentBespokeConditions` to `bespokeConditionsToDisplay`, but again, this is very minor so if you disagree, don't worry.

Reply:
updated the variable name from parentBespokeConditions to bespokeConditionsToDisplay.